Detectors/AOD/src/AODProducerWorkflowSpec.cxx

```diff
@@ -92,6 +92,13 @@ using SMatrix55Sym = ROOT::Math::SMatrix<double, 5, 5, ROOT::Math::MatRepSym<dou
 namespace o2::aodproducer
 {
 
+void AODProducerWorkflowDPL::CTPReadout(const o2::globaltracking::RecoContainer& recoData, gsl::span<o2::ctp::CTPDigit> ctpDigits)
```

Comment:
ctpDigits is what you fill when emulating the trigger, right? Then it should be a `std::vector<o2::ctp::CTPDigit> &ctpDigits`.
The rest is ok.

Reply:
Hi @shahor02 ,
yes, ctpDigita are filled.
I have three questions:
1)  can I get runnumber inside AODProducerWorkflowSpec ? ( I need it for getting CTPconfig from CCDB)
2)  to know if PHS and EMC are triggered I use 
const auto& caloEMCCellsTRGR = recoData.getEMCALTriggers();
const auto& caloPHOSCellsTRGR = recoData.getPHOSTriggers();
3.) what would be the simplest way to know if TRD is readout ?
Thanks, Roman.

Reply:
@Hi @shahor02 , no need to comment on my questions in previous post. With the help of @mpuccio it is answered. Cheers, R.

Reply:
Sorry, actually missed your message...

Reply:
@lietava Sorry, I see this only now, would have been good if you would have pinged me and the PHOS experts directly, I did not receive any type of notification. For EMCAL yes, we create one trigger record per event which is read out, independent of the payload size. Note that we correct the interaction record in the synchronous reconstruction by the L0_LM delay. Since the CTF  for EMCAL contains also the calibration triggers we have also the trigger type in our trigger record object, but since you correlate later with TVX this information is probably not needed.